Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -32,6 +32,57 @@ public class ProjectGenerator {
         return XcodeProj(workspace: workspace, pbxproj: pbxProject, sharedData: sharedData)
     }
 
+    public func generateFiles() throws {
+
+        /*
+         Default info plist attributes taken from:
+         /Applications/Xcode.app/Contents/Developer/Library/Xcode/Templates/Project Templates/Base/Base_DefinitionsInfoPlist.xctemplate/TemplateInfo.plist
+        */
+        var defaultInfoPlist: [String: Any] = [:]
+        defaultInfoPlist["CFBundleIdentifier"] = "$(PRODUCT_BUNDLE_IDENTIFIER)"
+        defaultInfoPlist["CFBundleInfoDictionaryVersion"] = "6.0"
+        defaultInfoPlist["CFBundleExecutable"] = "$(EXECUTABLE_NAME)"
+        defaultInfoPlist["CFBundleName"] = "$(PRODUCT_NAME)"
+        defaultInfoPlist["CFBundleDevelopmentRegion"] = "$(DEVELOPMENT_LANGUAGE)"
+        defaultInfoPlist["CFBundleShortVersionString"] = "1.0"
+        defaultInfoPlist["CFBundleVersion"] = "1"
```

Comment:
Could this be stored in a static property?

Reply:
It could indeed. I'll refactor this stuff out into a `InfoPlistGenerator` as well